lua/entities/rail_shell/init.lua

```diff
@@ -66,30 +66,31 @@ end
 
 				if tr.Hit then
 					if tr.HitSky then
-					self.Entity:Remove()
-					return true
+                        self.Entity:Remove()
+                        return true
 					end
 					if tr.MatType==83 then				//83 is wata
-					local effectdata = EffectData()
-					effectdata:SetOrigin( tr.HitPos )
-					effectdata:SetNormal( tr.HitNormal )		// In case you hit sideways water?
-					effectdata:SetScale( 60 )			// Big splash for big bullets
-					util.Effect( "watersplash", effectdata )
-					self.Entity:Remove()
-					return true
+                        local effectdata = EffectData()
+                        effectdata:SetOrigin( tr.HitPos )
+                        effectdata:SetNormal( tr.HitNormal )		// In case you hit sideways water?
+                        effectdata:SetScale( 60 )			// Big splash for big bullets
+                        util.Effect( "watersplash", effectdata )
+                        self.Entity:Remove()
+                        return true
 					end
 
 					local owner = IsValid( self:GetOwner() ) and self:GetOwner()
 					local inflictor = owner or self.Turret
 					util.BlastDamage(inflictor, self.Turret, tr.HitPos, 400, 100)
+                    util.BlastDamage(inflictor, self.Turret, tr.HitPos, 200, 300)
 
 					local effectdata = EffectData()
 					effectdata:SetOrigin(tr.HitPos)				// Position of Impact
 					effectdata:SetNormal(tr.HitNormal)			// Direction of Impact
 					effectdata:SetStart(self.flightvector:GetNormalized())	// Direction of Round
 					effectdata:SetEntity(self.Entity)			// Who done it?
 					effectdata:SetScale(2.2)				// Size of explosion
-					effectdata:SetRadius(tr.MatType)			// Texture of Impact
+					effectdata:SetRadius(67)			// Texture of Impact

```

Comment:
Can you assign `67` to a variable name that explains what it is?

Reply:
yes good spot, this is the "fix" for the constant errors that are always spammed by emplacements.
this effect throws an error if its radius isn't the mat type for concrete